### PR TITLE
Set mongoose dependency as gte

### PR DESCRIPTION
**Pull request by manoskouvarakis**
Palantir that is dependent to eip-mongo, downgrades mongoose version due to that dependency

### CODE REVIEW

**Review by nikostoulas — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Start MongoDB
+        uses: supercharge/mongodb-github-action@1.3.0
+        with:
+          mongodb-version: ${{ matrix.mongodb-version }}
+
+      - name: Start RabbitMQ
```
Comment (nikostoulas) [inline .github/workflows/test.yml]: ❓ is rabbitmq needed?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+npm-debug.log
+node_modules
+coverage
+build
```
Comment (nikostoulas) [inline .gitignore:4]: ❓ Is there an .npmignore? If not this will be used and build will not be included

Comment (manoskouvarakis) [reply .gitignore:4]: Forgot to add it. Done

File: .github/workflows/test.yml (hunk 1 above)
Comment (manoskouvarakis) [reply .github/workflows/test.yml]: Correct. Removed

**Pull request merged**